fixtures/python/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.black]
```

Comment:
Would it be possible to upgrade the test package to a more modern Python project structure (`pyproject.toml`-based to avoid upgrading in the future)?